plugins/updater/src/config.rs

```diff
@@ -113,3 +118,13 @@ impl<'de> Deserialize<'de> for UpdaterEndpoint {
         Ok(Self(url))
     }
 }
+
+fn deserialize_os_string<'de, D>(deserializer: D) -> Result<Vec<OsString>, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    Ok(Vec::<String>::deserialize(deserializer)?
+        .into_iter()
+        .map(OsString::from)
+        .collect::<Vec<_>>())
+}
```

Comment:
This is effectively the same as deserializing a String, let's just change `installer_args` field type to `Vec<String>`

Reply:
If we change this to `String`, do we want to also change the one from Builder to also be `String`?

Reply:
Also, I would like to open another PR to migrate current powershell implementation to `ShellExecute`, in that case I think `String` type makes more sense as we will convert it though `HSTRING::from` anyway

Reply:
> If we change this to `String`, do we want to also change the one from Builder to also be `String`?

yeah let's do it



> Also, I would like to open another PR to migrate current powershell implementation to `ShellExecute` to avoid flushing powershell window, in that case I think `String` type makes more sense as we will convert it though `HSTRING::from` anyway

`HSTRING` can convert from `OsString` as well but that's beside the point, `ShellExecute` sounds good to me.